docs/www/tls-kubernetes.md

```diff
@@ -0,0 +1,375 @@
+
+The Custom Resource Definition (CRD) of a Redpanda cluster includes three APIs: Kafka, Admin and Pandaproxy APIs. Each API can support TLS for server authentication and mTLS (Mutual TLS) for client authentication.
+
+TLS allows us to establish a secure connection between a client and a broker in which communication is encrypted. 
+
+Mutual TLS (mTLS), or 2-way TLS, is an authentication method in which a client attempting to establish a connection with the server (broker) must present its certificate. This way the server can decide whether the client can be trusted.
+
+## Getting started
+
+To demonstrate how to get up and running with TLS in Kubernetes, we’ll use a sample `yaml` configuration file for a one-node cluster.
+
+To enable TLS we need to add a listener entry under the API fields in our yaml configuration file, and then specify the subfield `tls` where we can enable TLS.
+
+For example, check that `kafkaApi` now have a new flag that says `tls: enabled: true`.
+
+Save this file as `yaml`:
+
+```yaml
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+ name: cluster-sample-tls
+spec:
+ image: "vectorized/redpanda"
+ version: "latest"
+ replicas: 1
+ resources:
+   requests:
+     cpu: 1
+     memory: 1.2G
+   limits:
+     cpu: 1
+     memory: 1.2G
+ configuration:
+   rpcServer:
+     port: 33145
+   kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+   pandaproxyApi:
+    - port: 8082
+      tls:
+        enabled: true
+   adminApi:
+   - port: 9644
+   developerMode: true
+```
+
+We’ll use the Kafka API to go through the TLS configuration, but you can use similar settings to set up TLS for the other APIs. For this guide, we assume external connectivity is disabled unless otherwise stated.
+
+The operator supports a single listener with TLS per API. Additionally, if you have two listeners for the same API and one of them has external connectivity enabled, the operator assumes that the latter listener has TLS enabled.
+
+Providing an issuer or certificate to the operator is only available for the Kafka API. 
+
+### Client Auth (mTLS)
+
+When you enable TLS, the client has the option of verifying the identity of the server, but to do this the client must have the CA’s certificate available in a trusted store.
+
+We can then enable client authentication (mTLS) on the same listener by setting `requireClientAuth` to `true`.
+
+So for example, we can edit our previous `yaml` file to add this new setting: 
+
+```yaml
+kafkaApi:
+    - port: 9092
+    tls:
+        enabled: true
+        requireClientAuth: true
+```
+
+When you enable client authentication, you must specify a key-pair (certificate and key). As we will later see, the operator can also generate the key-pair.
+
+When you finish setting up your configuration, apply it by running:
+
+```bash
+kubectl apply \
+-f <your-file-path.yaml>
+```
+
+Remember to change to the actual file path that you used. 
+
+
+## Managing Certificates
```

Comment:
Can we add a general sentence or two here so this heading isn't empty?

Reply:
```suggestion
## Managing Certificates

In this section, we discuss how the certificates are generated, how the renewal process looks like and how you can read or inspect your certificates
```
@coral-waters wdyt?

Reply:
Minor changes: 
In this section, we discuss how the certificates are generated, what the renewal process looks like, and how you can view your certificates.

Reply:
```suggestion
## Managing Certificates
In this section, we discuss how the certificates are generated, what the renewal process looks like, and how you can view your certificates.
```